<!DOCTYPE html>
<html lang="en-us">
  <head>
    <meta charset="UTF-8">
    <title>Community software tools for ngs by NCBI-Hackathons</title>
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <link rel="stylesheet" type="text/css" href="stylesheets/normalize.css" media="screen">
    <link href='https://fonts.googleapis.com/css?family=Open+Sans:400,700' rel='stylesheet' type='text/css'>
    <link rel="stylesheet" type="text/css" href="stylesheets/stylesheet.css" media="screen">
    <link rel="stylesheet" type="text/css" href="stylesheets/github-light.css" media="screen">
  </head>
  <body>
    <section class="page-header">
      <h1 class="project-name">Community software tools for ngs</h1>
      <h2 class="project-tagline"></h2>
      <a href="https://github.com/NCBI-Hackathons/Community_Software_Tools_for_NGS" class="btn">View on GitHub</a>
      <a href="https://github.com/NCBI-Hackathons/Community_Software_Tools_for_NGS/zipball/master" class="btn">Download .zip</a>
      <a href="https://github.com/NCBI-Hackathons/Community_Software_Tools_for_NGS/tarball/master" class="btn">Download .tar.gz</a>
    </section>

    <section class="main-content">
      <h3>
<a id="clinvar-tools-on-github" class="anchor" href="#clinvar-tools-on-github" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>ClinVar Tools on github</h3>

<p><a href="https://hermesfj.github.io/SNPs/" title="Metatools">Metatoolstest</a></p>

<p><a href="https://github.com/abramconnelly/clinvar-report">Abramconnelly</a>  </p>

<ul>
<li>Files parsed/used: xml and text</li>
<li>Output: Text custom for their workflow and TSV</li>
<li>Output columns: Chromosome, Position, Name, Significance, Frequency, Zygosity, ACC URL<br>
</li>
</ul>

<p><a href="https://github.com/agermanidis/ClinVar">Agermanidis</a></p>

<ul>
<li>A tool for transforming a XML datasource into JSON documents according to a specified schema. The tool lets you define a simple schema in a python dict (schema.py). --&gt; The script uses a simple DSL to specify the fields to extract from the input XML to produce the output.</li>
</ul>

<p><a href="https://github.com/alexhenrie/clinvar-for-nerds">Alexhnrie</a></p>

<ul>
<li>ClinVar for Nerds is a web-based interface to query ClinVar data and download it in CSV, JSON, VCF, JSON-LD (RDF model based JSON) format. The project setup downloads the clinvar full xml, parses it into json objects and dumps them in a mongodb instance. After that you can query the data in mongodb.
The web interface can take a file format, multiple "restriction" filters as inputs.</li>
</ul>

<p><a href="https://github.com/ClinGen/clinvar-demo">ClinGen</a></p>

<ul>
<li>Demonstration of the genboree.com REST API on a website using a React component to retrieve and display data.</li>
</ul>

<p><a href="https://github.com/clingendb/clinvar_xml_pipe">Clingendb</a>    </p>

<ul>
<li>The ClinVar database, maintained by the National Center for Biotechnology information (NCBI), provides information about medically important variants with associated phenotypes and effects of sequence changes. </li>
</ul>

<p><a href="https://github.com/dakl/macarthur-clinvar-to-vcf">Dakl</a>    </p>

<ul>
<li>The MacArthur lab has made a great effort make ClinVar data more available to the bioinformatics community. This script converts their flat file to a VCF file. </li>
</ul>

<p><a href="https://github.com/ianjw11/clinvarparser">Ianjw11</a>    </p>

<ul>
<li>to run converter script:
python run.py --source /path/to/clinvar.xml --output /path/to/output.json</li>
</ul>

<p><a href="https://github.com/macarthur-lab/clinvar">MacArthur</a>  </p>

<ul>
<li>This repo provides tools to convert ClinVar data into a tab-delimited flat file, and also provides that resulting tab-delimited flat file.</li>
</ul>

<p><a href="https://github.com/PersonalGenomesOrg/vcf2clinvar">PersonalGenomesOrg</a>     </p>

<ul>
<li>Generate a report based on matching a genome VCF to the VCF version of ClinVar's variant database. (possibly related to <a href="https://github.com/abramconnelly/clinvar-report">https://github.com/abramconnelly/clinvar-report</a>, same team etc.) This code is intended to be used as a module (installable via pip), for use in other systems or pipelines.</li>
</ul>

<p><a href="https://github.com/pidr/clinvar-rdfizer">Pidr</a>   </p>

<p><a href="https://github.com/rbagnall/clinvar_parser">Rbagnall</a>   </p>

<ul>
<li>Used to format Clinvar pathogenicity predictions into a snp hash table and indel hash table. Puts mutliple predictions for the same nucleotide into one line. Usage clinvar_parser.pl variant_summary.txt.</li>
</ul>

<p><a href="https://github.com/sephraim/convert_ClinVar_XML_to_VCF">Sephraim</a>     </p>

<ul>
<li>
</ul>

<p><a href="https://github.com/welchbm/VisualizeClinvar">Welchbm</a>        </p>

<hr>

<h3>
<a id="metagenomics-tools" class="anchor" href="#metagenomics-tools" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>Metagenomics Tools</h3>

<p><a href="http://pitgroup.org/amphoranet/">AmphoraNet</a><br>
<a href="http://clams.jgi-psf.org/">ClaMS</a><br>
<a href="http://clark.cs.ucr.edu/">Clark</a><br>
<a href="http://comet2.gobics.de/">CoMet-Universe</a><br>
<a href="https://www.ebi.ac.uk/metagenomics/">EBI Metagenomics</a><br>
<a href="http://viceroy.eeb.uconn.edu/EstimateS/">EstimateS</a><br>
<a href="https://github.com/biocore/Evident/">E-vident</a><br>
<a href="http://www.sysbio.se/Fantom/">Fantom</a><br>
<a href="http://ccb.jhu.edu/software/FLASH/">FLASh</a><br>
<a href="http://genomics1.mh-hannover.de/genometa/">Genometa</a><br>
<a href="http://www.cbcb.umd.edu/software/glimmer-mg/">Glimmer-MG</a><br>
<a href="http://lanl-bioinformatics.github.io/GOTTCHA/">GOTTCHA</a><br>
<a href="https://img.jgi.doe.gov/cgi-bin/m/main.cgi">IMG/M</a><br>
<a href="https://pods.iplantcollaborative.org/wiki/display/WiNGS/Scalable+metagenomic+analysis+using+iPlant">iPlant</a><br>
<a href="https://ccb.jhu.edu/software/kraken/">Kraken</a><br>
<a href="http://ab.inf.uni-tuebingen.de/software/megan6/">MEGAN6</a><br>
<a href="http://metaabc.iis.sinica.edu.tw/">MetaABC</a><br>
<a href="http://www.metagenassist.ca/METAGENassist/faces/Home.jsp">METAGENassist</a><br>
<a href="http://www.cbcb.umd.edu/software/metagenomeSeq">metagenomeSeq</a><br>
<a href="http://i.cs.hku.hk/%7Ealse/hkubrg/projects/metaidba/">Meta-IDBA</a><br>
<a href="http://meta.microbesonline.org/">metaMicrobesOnline</a><br>
<a href="https://github.com/marbl/metAMOS">MetAMOS</a><br>
<a href="http://huttenhower.sph.harvard.edu/metaphlan">MetaPhlAn</a><br>
<a href="http://metaphyler.cbcb.umd.edu/">MetaPhyler</a><br>
<a href="http://jcvi.org/metarep/">Metarep</a><br>
<a href="http://metavelvet.dna.bio.keio.ac.jp/">MetaVelvet</a><br>
<a href="http://metavir-meb.univ-bpclermont.fr/">METAVIR</a><br>
<a href="http://www.mothur.org/">Mothur</a><br>
<a href="http://enve-omics.ce.gatech.edu/mytaxa/">My Taxa</a><br>
<a href="https://www.onecodex.com/">One Codex</a><br>
<a href="https://github.com/neufeld/pandaseq">Panda</a><br>
<a href="http://www.exelixis-lab.org/pear">PEAR</a><br>
<a href="http://www.cbcb.umd.edu/software/phymm/">PhymmlBL</a><br>
<a href="http://qiime.org/install/virtual_box.html">QIIME</a><br>
<a href="http://edwards.sdsu.edu/RTMg/">Real Time Metagenomics</a><br>
<a href="http://rdp.cme.msu.edu/misc/about.jsp">Ribosomal Database Project</a><br>
<a href="http://www.bork.embl.de/software/smash/">SMASHCommunity</a><br>
<a href="https://omictools.com/taxator-tk-tool">Taxator-tk</a><br>
<a href="http://virome.dbi.udel.edu/">VIROME</a><br>
<a href="https://github.com/rdpstaff/Xander_assembler">Xander</a><br>
<a href="http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0030288">ZORRO</a>     </p>

<hr>

<h3>
<a id="other-tools" class="anchor" href="#other-tools" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>Other Tools</h3>

<p><a href="http://www.eecs.berkeley.edu/Pubs/TechRpts/2015/EECS-2015-159.pdf">SiRen</a><br>
<a href="http://www.well.ox.ac.uk/project-stampy">Stampy</a>         </p>

<hr>

<h3>
<a id="structural-variant-tools" class="anchor" href="#structural-variant-tools" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>Structural Variant Tools</h3>

<p><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3511991/">ERDS</a><br>
<a href="http://sebatlab.ucsd.edu/index.php/software-data">ForestSV</a><br>
<a href="https://github.com/PapenfussLab/gridss">GRIDSS</a><br>
<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3125812/">inGAP-sv</a><br>
<a href="https://github.com/arq5x/lumpy-sv">Lumpy</a><br>
<a href="https://github.com/InstitutCurie/SV-Bay">SV-Bay</a><br>
<a href="http://compbio.cs.sfu.ca/software-variation-hunter">VariationHunter</a>      </p>

<h3>
<a id="authors-and-contributors" class="anchor" href="#authors-and-contributors" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>Authors and Contributors</h3>

<p>You can <a href="https://help.github.com/articles/basic-writing-and-formatting-syntax/#mentioning-users-and-teams" class="user-mention">@mention</a> a GitHub username to generate a link to their profile. The resulting <code>&lt;a&gt;</code> element will link to the contributor’s GitHub Profile. For example: In 2007, Chris Wanstrath (<a href="https://github.com/defunkt" class="user-mention">@defunkt</a>), PJ Hyett (<a href="https://github.com/pjhyett" class="user-mention">@pjhyett</a>), and Tom Preston-Werner (<a href="https://github.com/mojombo" class="user-mention">@mojombo</a>) founded GitHub.</p>

<h3>
<a id="support-or-contact" class="anchor" href="#support-or-contact" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>Support or Contact</h3>

<p>Having trouble with Pages? Check out our <a href="https://help.github.com/pages">documentation</a> or <a href="https://github.com/contact">contact support</a> and we’ll help you sort it out.</p>

      <footer class="site-footer">
        <span class="site-footer-owner"><a href="https://github.com/NCBI-Hackathons/Community_Software_Tools_for_NGS">Community software tools for ngs</a> is maintained by <a href="https://github.com/NCBI-Hackathons">NCBI-Hackathons</a>.</span>

        <span class="site-footer-credits">This page was generated by <a href="https://pages.github.com">GitHub Pages</a> using the <a href="https://github.com/jasonlong/cayman-theme">Cayman theme</a> by <a href="https://twitter.com/jasonlong">Jason Long</a>.</span>
      </footer>

    </section>

  
  </body>
</html>
